@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -592,13 +592,16 @@ export class VisibleTileSet {
             maxElevation < EarthConstants.MAX_ELEVATION - maxBuildingHeight
                 ? maxElevation + maxBuildingHeight
                 : maxElevation;
```

Comment:
If buildings height is now included (accounted) in tile elevation, there is no need to add it here.... My proposal would be to define tile.maxGeometryHeight = EarthConstants.MAX_ELEVATION as default. Then before adding it to tileMaxElevation account for projection scale. 
So in simple words if decoder will not provide accurate value, we will use Abu Khalifa, if accurate height is delivered from decoder then we will have better accuracy and more fine tuned near plane position.